src/modules/ssh_apt/mappers/index.ts

```diff
@@ -0,0 +1,130 @@
+import { SshAptModule } from '..';
+import { Context, Crud, MapperBase } from '../../interfaces';
+import { Package } from '../entity';
+
+export class PackageMapper extends MapperBase<Package> {
+  module: SshAptModule;
+  entity = Package;
+
+  equals = (a: Package, b: Package) =>
+    Object.is(a.server, b.server) &&
+    Object.is(a.package, b.package) &&
+    Object.is(a.version, b.version) &&
+    Object.is(a.architecture, b.architecture) &&
```

Comment:
`server`, `package`, `version`, and `architecture` are annotated with `@cloudId` decorator, therefore when two objects are being checked for equality, they're already equal to each other.

Reply:
I wasn't sure exactly which columns would be part of the ID at first so I added everything, then never went back to clean this up.